machine/meta-rcar-gen3/recipes-graphics/gles-module/gles-user-module.bbappend

```diff
@@ -0,0 +1,72 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+COMPILER_URL = "${TOPDIR}/../proprietary/rcar/meta/META_Embedded_Toolkit_2.8.1.0.3.1.zip"
+
+SRC_URI_remove = "file://change-shell.patch \
+                  file://0001-EGL-eglext.h-Include-eglmesaext.h-to-avoid-compile-error.patch \
+"
+SRC_URI_append = "file://${COMPILER_URL} \
+                  file://pkgconfig/egl.pc \
+                  file://pkgconfig/glesv2.pc \
+                  file://rcpvr-change-shell.patch\
+                  file://0001-EGL-eglext.h-Include-eglmesaext.h.patch \
+"
+DEPENDS += "virtual/kernel wayland-kms libgbm mesa"
+
+EXTRA_OEMAKE += "CROSS_COMPILE=${TARGET_PREFIX}"
+EXTRA_OEMAKE += "PVR_BUILD_DIR=${PVRUM_BUILD_DIR}"
+EXTRA_OEMAKE += "DISCIMAGE=${PVRUM_DISCIMAGE}"
+EXTRA_OEMAKE += "KERNELDIR=${STAGING_KERNEL_BUILDDIR}"
+EXTRA_OEMAKE += "METAG_INST_ROOT=${S}/metag/2.8"
+
+# Need to actually delete the varflag not just unset it.
+python __anonymous() {
+    d.delVarFlag('do_compile', 'noexec')
+}
+
+# Since we depend on wayland-kms we have to provide
+# following header files to make it buildable.
+do_populate_headers() {
```

Comment:
instead of introducing one mare task can we have do_patch_append here?
it will run after do_patch which seems to be your goal

Reply:
Done